R/demo-crowding.R

```diff
@@ -39,14 +45,17 @@
 #'               race = "black")
 #' }
 #' 
-demo_crowding <- function(area, areatype, crowding, race) {
+demo_crowding <- function(area, areatype, 
+                          crowding = "household with >1 person per room", 
+                          race) {
 
+  areatype <- tolower(areatype)
```

Comment:
Doing this up front because we need the lowercase three times: For the resp, for the `areatype_map`, and for the `areacode_map`.